lib/contracts/accountParser.js

```diff
@@ -0,0 +1,65 @@
+const bip39 = require("bip39");
+const hdkey = require('ethereumjs-wallet/hdkey');
```

Comment:
There a particular reason for using both of these?

Reply:
It's to get the private keys from the mnemonic. Web3 doesn't seem to have the ability to do so

Reply:
right, but why use both? Why not just one or the other? IIRC they both do the same thing.

Reply:
Whoops. Nvm, carry on.